src/configuration.ts

```diff
@@ -44,14 +45,15 @@ export class RLSConfiguration {
      * directory.
      */
     public readonly rlsRoot: string | null;
-
+    
```

Comment:
nit: please don't add whitespace

Reply:
Ok, I will correct this and pay attention to this in future.